app/library/postgres.py

```diff
@@ -72,9 +73,13 @@ async def get_asset_from_postgres(asset_id: str):
 
 async def delete_asset_from_postgres(asset_id: str):
     # TODO: delete from postgres
+    try:
+        delete_asset(asset_id)
+    except Exception as e:
+        raise e
 
-    sample_response = {"message": "deleted successfully"}
-    return sample_response
+    response = {"message": "deleted successfully"}
```

Comment:
If an asset doesn't exist in the table, this method still returns `{"message": "deleted successfully"}`. Should we do a check to see if the asset exists first? Or is there a better way of distinguishing between non-valid asset IDs and valid asset IDs that are deleted?

Reply:
We shouldn't really allow the deletion of non-valid asset IDs in the frontend UI. But either way, I agree a different error message should be returned. I'll look into that